lib/util/generate_strategy_results.js

```diff
@@ -20,7 +20,9 @@ const formatPositions = (positions = []) => {
   }
 
   return Object.fromEntries(positions.map(p => {
-    const { id, entryPrice, entryAt, closingPrice, closedAt, pl, symbol, trades } = p
+    const { id, entryPrice, entryAt, closingPrice, closedAt, pl, symbol, trades = [] } = p
+    const amount = trades.reduce((acc, t) => acc + t.amount, 0)
```

Comment:
Amount is being updated on every new trades placed for that position.. We can directly read amount from the position object. But this works as well.